core/src/main/java/org/apache/cxf/common/util/SpringClasspathScanner.java

```diff
@@ -170,7 +172,7 @@ private static boolean isSpringInOsgi() {
 
     private ResourcePatternResolver getResolver(ClassLoader loader) {
         ResourcePatternResolver resolver = null;
-        if (IN_OSGI) {
+        if (IN_OSGI != null && IN_OSGI.booleanValue()) {
```

Comment:
use primitive boolean

Reply:
@amarkevich . Done with thanks.  Great  idea from you.